gremlin-go/driver/connectionPool_test.go

```diff
@@ -35,8 +35,7 @@ var logger = newLogHandler(&defaultLogger{}, Info, language.English)
 func getPoolForTesting() *loadBalancingPool {
 	return &loadBalancingPool{
 		url:                    "",
-		authInfo:               nil,
-		tlsConfig:              nil,
+		connSettings:           newDefaultConnectionSettings(),
```

Comment:
Do we need to maintain nil for tests? I can update this to make `authInfo` and `tlsConfig` in `connSettings` nil if needed but the tests seem to pass.